Source/Scene/Cesium3DTile.js

```diff
@@ -455,7 +499,69 @@ define([
      * @private
      */
     Cesium3DTile.prototype.requestContent = function() {
-        this._content.request();
+        var that = this;
+
+        if (this.hasEmptyContent) {
+            return false;
+        }
+
+        if (!this.canRequestContent()) {
+            return false;
+        }
+
+        var distance = this.distanceToCamera;
+        var promise = RequestScheduler.schedule(new Request({
+            url : this._contentUrl,
+            server : this._requestServer,
+            requestFunction : loadArrayBuffer,
+            type : RequestType.TILES3D,
+            distance : distance
+        }));
+
+        if (!defined(promise)) {
+            return false;
+        }
+
+        this._contentState = Cesium3DTileContentState.LOADING;
+        this._contentReadyToProcessPromise = when.defer();
+        this._contentReadyPromise = when.defer();
+
+        promise.then(function(arrayBuffer) {
+            if (that.isDestroyed()) {
+                return when.reject('tileset is destroyed');
+            }
+            var uint8Array = new Uint8Array(arrayBuffer);
+            var magic = getMagic(uint8Array);
+            var contentFactory = Cesium3DTileContentFactory[magic];
+            var content;
+
+            if (defined(contentFactory)) {
+                content = contentFactory(that._tileset, that, that._contentUrl, arrayBuffer, 0);
+                that.hasRenderableContent = true;
+            } else {
+                // The content may be json instead
```

Comment:
In the case when the tileset is JSON, do you know if the overhead here is high?  Perhaps not since `new Uint8Array` should be basically free?  It's important that we do not create high overhead for tilesets of tilesets since I expect many users will use this feature for large tilesets.